src/shared/lib/returns/badge.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const titleCase = require('title-case');
+
+/**
+ * Gets badge object to render for return row
+ * @param  {String}  status    - return status
+ * @param  {Boolean} isPastDue - whether return is past due
+ * @return {Object}            - badge text and style
+ */
+const getBadge = (status, isPastDueDate) => {
+  const viewStatus = ((status === 'due') && isPastDueDate) ? 'overdue' : status;
+
+  const styles = {
```

Comment:
I have recently updated the badge classes in another PR:
https://github.com/DEFRA/water-abstraction-ui/pull/975